test/wh_test_crypto.c

```diff
@@ -340,6 +417,16 @@ int whTest_CryptoServerConfig(whServerConfig* config)
             break;
         }
         wh_Server_GetConnected(server, &am_connected);
+        /* keep alive for 2 user changes */
+        if (am_connected != WH_COMM_CONNECTED && userChange < 2) {
+            if (userChange == 0)
+                server->comm->client_id = 2;
```

Comment:
I would remove this and simply use the wh_Client_CommInit function to update the server-side state.